meson.build

```diff
@@ -788,7 +792,7 @@ else
 endif
 
 if with_quota != 'false' and not have_quota
-    message('Ouota support requested but libtirpc or libquota libraries not found')
+    warning('Ouota support requested but libtirpc or libquota libraries not found')
```

Comment:
Typo: The first word of the string should be "Quota" with a Q

Reply:
@rdmark, Oops missed this one! I'll correct it in the options semantics PR I'm preparing for v2...